.github/workflows/build.yaml

```
@@ -42,7 +42,13 @@ jobs:
     needs: setup
     strategy:
       matrix:
-        runner: [windows-large, macos-15-xlarge]
+        runner: ${{ 
+          fromJson(
+            github.ref_type == 'tag' && startsWith(github.ref, 'refs/tags/Second_Life')
+            ? '["windows-large", "macos-15-xlarge"]'
+            : '["windows-latest", "macos-15"]'
+          ) 
+        }}
```

```suggestion
        runner: ${{
          fromJson(
            github.ref_type == 'tag' && startsWith(github.ref, 'refs/tags/Second_Life')
            ? '["windows-large", "macos-15-xlarge"]'
            : '["windows-latest", "macos-15"]'
          )
        }}
```

Sorry, I forgot to clean up the trailing spaces.